bazel/repository_locations.bzl

```diff
@@ -721,10 +721,10 @@ REPOSITORY_LOCATIONS_SPEC = dict(
     rules_python = dict(
         project_name = "Python rules for Bazel",
         project_desc = "Bazel rules for the Python language",
-        project_url = "https://github.com/bazelbuild/rules_python",
-        version = "0.8.1",
-        sha256 = "cdf6b84084aad8f10bf20b46b77cb48d83c319ebe6458a18e9d2cebf57807cdd",
-        release_date = "2022-04-22",
+        project_url = "https://github.com/phlax/rules_python",
+        version = "7cd86e2dcc87dced66336fc684777df453391915",
+        sha256 = "fa6511b7ff9cc949a5bce5a4a1d628488151309f6e06b3997e5dfccba1ecdc79",
+        release_date = "2022-05-18",
```

Comment:
If we go with this workaround, please note which release of rules_python is expected to pick up the fix upstream, and leave the canonical origin url commented out

Reply:
we dont need to - it was just me testing

it now works with the published version - there is an upstream PR to moved to the shared-msvc - and i think that also now works - i will test it before i deWIP this PR

needing to compile makes the issue kinda moot, at least for now tho, so i think we should be good to land with released version